tests/unit/core/test_api_errors.py

```
@@ -0,0 +1,789 @@
+        if message["type"] == "http.response.body"
+    ) == prefix + oversized_chunk + suffix
+
+
+@pytest.mark.asyncio
+async def test_oversized_error_json_is_fail_closed_not_flushed():
+    sent_messages = []
+    prefix = b'{"detail":"'
+    oversized_chunk = b"x" * (64 * 1024 + 1)
+    suffix = b'"}'
+
+    async def app(scope, receive, send):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `app`

Add return type annotation: `None`

(ANN202)